test/functional/wallet_importdescriptors.py

```diff
@@ -705,6 +706,13 @@ def run_test(self):
             except JSONRPCException as e:
                 assert e.error["code"] == -4 and "Error: the wallet is currently being used to rescan the blockchain for related transactions. Please call `abortrescan` before changing the passphrase." in e.error["message"]
 
+            wallet_info = self.nodes[0].cli("-rpcwallet=encrypted_wallet").getwalletinfo()
+            try:
+                duration = wallet_info["scanning"]["duration"]
+                assert_greater_than(duration, 0)
+            except Exception:
+                assert "scanning" not in wallet_info
```

Comment:
nit: Seems a bit heavy-handed to use exceptions, even if it's following the pattern of code above?
```suggestion
            scanning = wallet_info.get("scanning")
            if scanning:
                assert_greater_than(scanning["duration"], 0)
```
Could also break out `encryped_cli = self.nodes[0].cli("-rpcwallet=encrypted_wallet")` for the entire block now that we are using it for a 4th time, if you re-touch.